setup-client-usbdrive

```diff
@@ -35,7 +35,10 @@ if not get_consent():
 
 # create partition table and write MBR
 command('sfdisk', args.blockdev, stdin="label: dos\n2048,1638400,c,*\n")
-command('dd', 'if=/usr/lib/syslinux/bios/mbr.bin', 'of=' + args.blockdev)
+try:
+    command('dd', 'if=/usr/lib/syslinux/bios/mbr.bin', 'of=' + args.blockdev)
+except:
+    command('dd', 'if=/usr/lib/syslinux/mbr/mbr.bin', 'of=' + args.blockdev) #debian
```

Comment:
it'd be better to search for the MBR file first, or determine the path depending on `cfg.modules`, and then call `dd` once